src/pages/MyMeetingPages/RegisterDetailPage.tsx

```diff
@@ -0,0 +1,54 @@
+import $ from './style.module.scss';
+import { useState } from 'react';
+import {
+  registerMeetingMocks,
+  requestListMocks,
+} from 'src/__mocks__/myMeeting';
+import { RequestedList } from 'src/components/MyMeeting';
+import { InfiniteScroll } from 'src/components/Layout';
+import { MyMeetingRequestType } from 'src/types/myMeeting';
+
+const { title, content, friends, date } = registerMeetingMocks[0];
+
+export default function RegisterDetailPage() {
+  const [requestedMeeting, setRegisterMeeting] = useState<
+    MyMeetingRequestType[]
+  >([]);
+
+  const fetchMoreMeetingFeeds = () => {
+    setRegisterMeeting([...requestedMeeting, ...requestListMocks]);
+  };
+
+  return (
+    <>
+      <div className={$['detail-info']}>
+        <div className={$['friends-image-list']}>
+          {friends.map((imgUri, index) => (
+            <img key={`${imgUri}-${index}`} src={imgUri} alt="friends-image" />
+          ))}
+        </div>
+
+        <div className={$['info']}>
+          <div>
+            <span className={$['title']}>{title}</span>
+            <span className={$['date']}>•{date}</span>
+          </div>
+          <span className={$['content']}>{content}</span>
+        </div>
+      </div>
+
+      <InfiniteScroll trigger={fetchMoreMeetingFeeds}>
+        <ul>
+          {requestedMeeting.map(({ comment, profileImg, date }, index) => (
+            <RequestedList
+              key={`requested-list-${index}`}
+              comment={comment}
+              profileImg={profileImg}
+              date={date}
```

Comment:
여기를 전개 연산으로 풀어도 편할 것 같습니다~
```suggestion
              {...{comment, profileImg, date}}
```

Reply:
오호 좋습니다 !